README.md

```diff
@@ -2,3 +2,56 @@
 
 ## Getting started
 
+
+### At first you will need to install any git based controlsystem.
+
+[Github Desktop](https://desktop.github.com/) "graphical interface" 
+or 
+[Git](https://git-scm.com/downloads) "code  interface"<hr>
+### And the LTS version of NodeJS
+[NodeJS](https://nodejs.org/en)
+<hr>
+
+### A prefarable additional Download would be:
+<hr>
+
+A source-code editor like  [Visual Studio Code ](https://code.visualstudio.com/) 
+ And a browser with Bootstrap 5 support See if your borser is supported [here](https://github.com/twbs/bootstrap/blob/v5.0.2/.browserslistrc).
+<br>
+
+After downloading. Go to **https://github.com/stetsche/bootstrap-playground** and fork  this repository. If you do not know how this works, [here ](#forks) is an explanation on how to do it.
+After its finished you can start working on your own version of [bootstrap-playground](https://github.com/stetsche/bootstrap-playground)
+
+ If you want to publish changes on [bootstrap-playground](https://github.com/stetsche/bootstrap-playground) make a [pull request](#pull-request).
+
+### Forks
+<details>
+<summary> open fork explanation  </summary>
+<br>
+Go to https://github.com/stetsche/bootstrap-playground.<br>
+Click the "Fork" button on the top-right corner.<br>
+Wait for the fork to be created.<br>
+Clone the forked repository to your local machine.<br>
+Make changes to the code.<br>
+Commit and push your changes to the forked repository.<br>
+Optionally, create a pull request to contribute your changes back to the original repository.
+</details><hr>
+
+
+### Pull request
+<details>
+<summary> open pull request explanation  </summary>
+<br>
+Fork the repository.<br>
+Create a branch for your changes.<br>
```

Comment:
I think we don't need the <br> tags, you can leave a blank line as well. But in this case it would be nice to add them as a list:
1. Fork the repository.
2. Create a branch for your changes.
3. ....